notebooks/Shuvomoy - Benders decomposition.ipynb

```diff
@@ -407,12 +418,12 @@
     "@defVar(subProblemModel, u[1:dimU]>=0)\n",
     "    \n",
     "\n",
-    "@addConstraint(subProblemModel, constrRefSubProblem[j=1:size(A2,2)],sum{A2[i,j]*u[i] , i in [1:size(A2,1)]}>=c2[j])\n",
+    "    @addConstraint(subProblemModel, constrRefSubProblem[j=1:size(A2,2)],sum{A2[i,j]*u[i] , i in collect(1:size(A2,1))}>=c2[j])\n",
```

Comment:
Same concatenation issue.

On Mon, Jan 25, 2016 at 6:24 PM, Miles Lubin <notifications@github.com>
wrote:

> In notebooks/Shuvomoy - Benders decomposition.ipynb
> <https://github.com/JuliaOpt/juliaopt-notebooks/pull/17#discussion_r50773652>
> :
>
> > @@ -407,12 +418,12 @@
> >      "@defVar(subProblemModel, u[1:dimU]>=0)\n",
> >      "    \n",
> >      "\n",
> > -    "@addConstraint(subProblemModel, constrRefSubProblem[j=1:size(A2,2)],sum{A2[i,j]*u[i] , i in [1:size(A2,1)]}>=c2[j])\n",
> > +    "    @addConstraint(subProblemModel, constrRefSubProblem[j=1:size(A2,2)],sum{A2[i,j]*u[i] , i in collect(1:size(A2,1))}>=c2[j])\n",
>
> For example, this could just be i in 1:size(A2,1)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaOpt/juliaopt-notebooks/pull/17/files#r50773652>.
>



-- 
Shuvomoy Das Gupta
Ph.D. Student
Systems Control Group
Department of Electrical & Computer Engineering
University of Toronto
http://scg.utoronto.ca/~shuvomoy.dasgupta/